src/components/common/ProfilePanel.js

```diff
@@ -26,58 +34,57 @@ import Switch from './Switch';
  * togglePanel: function to be called when the panel is collapsed or opened
  */
 
-class ProfilePanel extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      nameIsHovering: false,
-      imageIsHovering: false,
-      editingName: false,
-      showModal: false,
-      nameSubmitted: false,
-      name: this.props.displayName,
-      selectedImage: '',
-      displayNameMessage: '',
-    };
-  }
-
-  handleOpenModal = () => {
-    this.setState({ showModal: true, selectedImage: this.props.photoName });
+function ProfilePanel(props) {
+  //States
+  const [nameIsHovering, setNameHover] = useState(false);
+  const [imageIsHovering, setImageHover] = useState(false);
+  const [editingName, setEditingName] = useState(false);
+  const [showModal, setShowModal] = useState(false);
+  const [nameSubmitted, setNameSubmitted] = useState(false);
+  const [name, setName] = useState(props.displayName);
+  const [selectedImage, setSelectedImage] = useState('');
+  const [displayNameMessage, setDisplayNameMessage] = useState('');
+  const [error, setError] = useState('');
+
```

Comment:
nit: let's destructure the props so that we know what props are actually used here. It also helps other know what props are required when using this component.

```
const {displayName, setDisplayName, ...} = props
```
and then we can use `displayName` instead of `props.displayName`